src/tools/tests/storage-key-recipe-resolver-test.ts

```diff
@@ -374,8 +374,7 @@ describe('recipe2plan', () => {
 
       const resolver = new StorageKeyRecipeResolver(manifest);
       for (const it of (await resolver.resolve())) {
-        assert.isTrue(it.isResolved());
-      }
+        assert.isTrue(it.isResolved()); }
```

Comment:
is this change intentional? seems wrong, `}` should go on the next line

Reply:
I applied an auto-formatter; I'm surprised it did such a weird thing.